sale_order_invoice_amount/hooks.py

```diff
@@ -0,0 +1,92 @@
+def _add_new_columns(cr):
+    cr.execute(
+        """
+ALTER TABLE
+    sale_order
+ADD COLUMN IF NOT EXISTS
+    invoiced_amount numeric,
+ADD COLUMN IF NOT EXISTS
+    uninvoiced_amount numeric
```

Comment:
```suggestion
    amount_invoiced numeric,
ADD COLUMN IF NOT EXISTS
    amount_to_invoice numeric
```

If change the field in sale.order, you have change here the field names. Fields invoiced_amount, uninvoiced_amount no exist.
Change the field names in all script

Reply:
Right now these fields are added by odoo, so I removed the hook that initializes them!